homeassistant/helpers/device_registry.py

```diff
@@ -369,7 +369,7 @@ async def _async_migrate_func(
                 for device in old_data["devices"]:
                     device["serial_number"] = None
             if old_minor_version < 5:
-                # Introduced in 2022.10
+                # Introduced in 2023.12
```

Comment:
Best Christmas gift this year! 🎅

Reply:
It doesn't mean anything. I've just updated the PR.

Reply:
This would be a game changing feature for me (and others too I’m sure). How can we help get this merged?

(Sorry for the noise, not sure if there’s a forum post for this)